lib/api/CInferenceModelDefinition.cc

```diff
@@ -380,6 +386,41 @@ std::string CInferenceModelDefinition::jsonString() {
     return resultString;
 }
 
+std::string CInferenceModelDefinition::jsonStringCompressedFormat() {
```

Comment:
Does this also split up the compressed string into a specified length?

I think we need to make guarantees around String length size so we don't ever reach the situation where even the compressed string is too large. 

This will prevent us from hitting Elasticsearch limits in:

- maximum over the wire document lengths
- maximum document size
- OOM issues for large Java strings

Reply:
You are right, I missed the chunking part so far.